src/js/pensions/config/form.js

```diff
@@ -113,6 +114,25 @@ function usingDirectDeposit(formData) {
   return formData['view:noDirectDeposit'] !== true;
 }
 
+function servedDuringWartime(period) {
+  const warDates = [
```

Comment:
I wonder if we should move this out of the function, so it's not created and destroyed on every call.